modules/manage/pages/manage-security/manage-tls.adoc

```diff
@@ -0,0 +1,160 @@
+
+[#set-the-minimum-tls-version-with-the-cli]
+=== Set the Minimum TLS Version with the CLI
+
+To set the minimum TLS version with the CLI, use the xref:cli:cbcli/couchbase-cli-setting-security.adoc[setting-security] command as follows:
+
+----
+/opt/couchbase/bin/couchbase-cli setting-security \
+-c 10.143.192.101:8091 \
+-u Administrator \
+-p password \
+--set \
+--tls-min-version tlsv1.1
+----
+
+The `set` flag indicates that a value is to be set.
+The `tls-min-version` flag specifies the appropriate minimum TLS value, which can be `tlsv1`, `tlsv1.1`, or `tlsv1.2`; and is in this case specified as `tlsv1.1`.
+
+If successful, the command returns the following success-message:
+
+----
+SUCCESS: Security settings updated
+----
+
+For more information, see xref:cli:cbcli/couchbase-cli-setting-security.adoc[setting-security]
+
+[#set-the-minimum-tls-version-with-the-rest-api]
+=== Set the Minimum TLS Version with the REST API
+
+To set the minimum TLS version with the REST API use the `POST /settings/security` http method and URI, as follows:
+
+----
+curl  -u Administrator:password -v -X POST \
+http://10.143.192.101:8091/settings/security \
+-d 'tlsMinVersion=tlsv1.2'
+----
+
+The `tlsMinVersion` flag specifies the minimum TLS version to be used; which can be `tlsv1`, `tlsv1.1`, or `tlsv1.2`; and is in this case specified as `tlsv1.2`
+If successful, the command gives a `200 OK` message, and returns an empty array.
+
+For more information, see xref:rest-api:rest-setting-security.adoc[Manage Encryption Settings].
+
+[#manage-cipher-suite-lists]
+== Manage Cipher-Suite Lists
+
+As described above, in xref:manage:manage-security/manage-tls.adoc#tls-and-cipher-suites[TLS and Cipher-Suites], a _custom_ cipher-suite list can be defined for Couchbase Server.
+Additionally, Couchbase Server can be configured to conform to either its own or the client's preference, in selecting a commonly supported cipher-suite to be used for the communication.
+These tasks can be accomplished with either the CLI or the REST API, as described below.
+
+[#manage-ciphers-with-the-cli]
+=== Manage Cipher-Suites with the CLI
+
+To manage cipher-suites with the CLI, use the xref:cli:cbcli/couchbase-cli-setting-security.adoc[setting-security] command as follows:
+
+----
+/opt/couchbase/bin/couchbase-cli setting-security \
+-c 10.143.192.101:8091 \
+-u Administrator \
+-p password \
+--set \
+--tls-honor-cipher-order 1 \
+--cipher-suites TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA
+----
+
+The `set` parameter establishes use of the command to make settings.
+The `tls-honor-cipher-order` parameter-value is specified as `1`, meaning that the server's preference-order for cipher-suites will be used, rather than the client's.
+(Note, however, that this setting is also the default; and so will remain established even if this parameter is not specified.)
+The `cipher-suites` parameter takes a value that is a list of cipher-suites to be used for the cluster.
```

Comment:
1) It is worth adding that command accepts IETF RFC names for cipher suites (there two naming standards: IETF RFC and OpenSSL).  
2) Please also add that empty list [] means that ciphers are not set (default value). It means that default set of ciphers will be used.

Reply:
I've made the comment about naming standards at the introduction to the section (I'm assuming it applies both the REST and CLI). I've added the empty list for both REST and CLI as [] and "" respectively.

Reply:
Sorry for the confusion. I meant that we only support IETF RFC names. We don't accept OpenSSL names.

Reply:
ok